smoketest.bash

```diff
@@ -205,6 +208,22 @@ if [ "${USE_PROXY}" = "true" ]; then
     createProxyCfgVolume
 fi
 
+createProxyCertsVolume() {
+    "${container_engine}" volume create auth_proxy_certs
+    "${container_engine}" container create --name proxy_certs_helper -v auth_proxy_certs:/certs busybox
+    if [ -f "${DIR}/compose/auth_certs/certificate.pem" ] && [ -f "${DIR}/compose/auth_certs/private.key" ]; then
+        chmod 644 "${DIR}/compose/auth_certs/private.key"
```

Comment:
```
$ ls -l private.key certificate.pem 
-rw-r--r--. 1 work work 1931 May  2 16:03 certificate.pem
-rw-------. 1 work work 3272 May  2 16:03 private.key
```

It looks like the default permissions right after running the script to create these files are `644` and `600`. Why is `644` needed? Not that it really matters since this key is only used for this test anyway, but would `640` also work?

Reply:
yes 640 also works